x/oracle/types/message_oracle.go

```diff
@@ -28,6 +30,23 @@ func (m *MsgRegisterOracle) Type() string {
 }
 
 func (m *MsgRegisterOracle) ValidateBasic() error {
```

Comment:
I think the followings should also be validated here.
- unique ID
- oracle address (validity)
- oracle commission rate